private/release/commit-and-open-pr.js

```diff
@@ -22,4 +22,5 @@ export default async function commit (spawnOptions, ...files) {
                  || `git@github.com:${REPO_OWNER}/${REPO_NAME}.git`
 
   console.log(`Please run \`git push ${remote} ${sha}:refs/heads/release\`.`)
+  console.log('Soon, a Release PR should open on GitHub. Do not merge it manually! Review the PR thoroughly, and if everything looks good to you, approve the PR.')
```

Comment:
```suggestion
  console.log(
      'Once you have pushed to the remote branch, an automation will kick ' +
      'off and open a release candidate PR on the GitHub repository. Do not ' +
      'merge it manually! Review the PR thoroughly, you may close and ' +
      're-open it so the CI can run on it. If everything looks good to you, ' +
      'approve the PR, do not merge it manually, it will be automatically ' +
      'merged after the new versions have been automatically published to ' +
      'the npm registry.'
    )
```

Reply:
Re-reading this, that level of details might be a bit overwhelming, maybe we can find a middle ground